src/utils/LinearAlg2D.cpp

```diff
@@ -200,13 +202,23 @@ coord_t LinearAlg2D::getDist2FromLine(const Point& p, const Point& a, const Poin
     const Point vap = p - a;
     const coord_t ab_size2 = vSize2(vab);
     const coord_t ap_size2 = vSize2(vap);
+    coord_t ax_size2, px_size2;
```

Comment:
This is more a matter of taste, but we only need `px_size2` outside if the scopes below. Just write `const coord_t ax_size2 = ...` twice below, and you're not really duplicating code and get rid of something that might be confusing here.

Defining multiple variables like this is also frowned upon a bit, because you
 a) leave them undefined,
 b) can do things like `int a, b[10], c;`.